src/dialogsettings.ui

```diff
@@ -327,6 +327,9 @@
               </item>
               <item>
                <widget class="QPushButton" name="btnAccountRemove">
+                <property name="toolTip">
+                 <string>You can also remove folder with keystroke on Delete or Backspace key</string>
```

Comment:
You might not have the necessary Qt package (`Qt5LinguistTools`) installed, but since this adds a new translatable string we should add it so it adds an entry to the translations. I can do this afterwards once this is merged.

Reply:
I hadn't seen the various translation files and had just looked at dynamic_*.ts :x

Translation added in a83c2ddd1b698996b1f5f13883af683a3492223c (For fr and en)

Reply:
Perfect, thanks for adding the translations!

Reply:
Ah, and thank you for adding the other unfinished translations as well! ❤️